.circleci/config.yml

```diff
@@ -16,19 +16,42 @@ workflows:
   continuous-integration:
     jobs:
       - python-checks:
+          matrix:
+            parameters:
+              python_version:
+                - "3.8"
+                - "3.9"
+                - "3.10"
+                - "3.11"
+          python_version: << matrix.python_version >>
           filters:
             branches:
               ignore: develop
           prj_dir: aineko
-          python_version: << pipeline.parameters.python_version >>
+          # python_version: << matrix.python_version >>
```

Comment:
Can we remove the commented lines?

Reply:
Addressed in 37d7433a8df88bc56f2ebfe921d5753191d1d321